bolt/bolt-protocol-message-specification-4.md

```diff
@@ -81,7 +81,7 @@ Protocol errors are fatal and should immediately transition the server state to
 
 ## Message Exchange
 
-Messages are exchanged in a request-response pattern between client and server.
+Messages are exchanged in a request-response pattern between the client and server.
```

Comment:
Both is proper English. If you prefer the definite form (with article) I'd argue to apply it to both nouns: "the server and the client".

Personally, I prefer the version without articles.

Reply:
> Both is proper English.
The spellchecker of IntelliJ does not agree :)